modules/extension/wmts/src/main/java/org/geotools/ows/wmts/model/WMTSCapabilities.java

```diff
@@ -370,13 +373,29 @@ private WMTSLayer parseLayer(LayerType layerType) {
             style.setName(styleType.getIdentifier().getValue());
             StringBuilder t = new StringBuilder();
             for (Object title1 : styleType.getTitle()) {
-                t.append(title1.toString());
+                if (title1 instanceof LanguageStringType) {
+                    t.append(((LanguageStringType) title1).getValue());
+                } else {
+                    t.append(title1.toString());
+                }
+            }
+
+            if (!t.isEmpty()) {
```

Comment:
this causes a compiler error, the `isEmpty()` method was added in Java 15: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/CharSequence.html#isEmpty()